src/V3EmitC.cpp

```diff
@@ -997,6 +997,25 @@ class EmitCStmts : public EmitCBaseVisitor {
         emitOpName(nodep, "VL_STREAML_%nq%lq%rq(%nw,%lw,%rw, %P, %li, %ri)", nodep->lhsp(),
                    nodep->rhsp(), NULL);
     }
+    virtual void visit(AstCountBits* nodep) {
+        putbs("VL_COUNTBITS_");
+        emitIQW(nodep->lhsp());
+        puts("(");
+        if (nodep->lhsp()->isWide()) {
```

Comment:
Be sure to cover by testing the data being < 32 bits, 33-63 bits, and > 65 bits.